app/views/state_file/questions/md_spouse_state_id/edit.html.erb

```diff
@@ -0,0 +1,24 @@
+<% title = t("state_file.questions.spouse_state_id.edit.title_html") %>
+<% content_for :page_title, title %>
+
+<%# Must be above :card content_for %>
+<% content_for :info_box do %>
+  <div class="reveal">
+    <p><a href="#" class="reveal__link"><%= t(".why_ask_this") %></a></p>
+    <div class="reveal__content">
+      <div class="with-top-separator">
+        <p>
+          <%=  t('.protect_identity') %>
+        </p>
+        <p>
+          <%= t('.encourage_sharing') %>
+        </p>
+      </div>
+    </div>
+  </div>
+<% end %>
+
+<% content_for :card do %>
+  <h1 class="h2"><%= title %></h1>
+  <%= render 'state_file/questions/ny_primary_state_id/state_id', options: {} %>
```

Comment:
since this form is used for all states, can we think about moving this into a more general state-specific folder?